src/test/java/net/masterthought/cucumber/StepTest.java

```diff
@@ -128,8 +128,9 @@ public void shouldCreateLinkToScreenshotWhenOneExists() throws IOException {
-                        + "<a onclick=\"attachment=document.getElementById('embedding-1234'); attachment.style.display = (attachment.style.display == 'none' ? 'block' : 'none');return false\" href=\"#\">Attachment 3 (plain text)</a><pre id=\"embedding-1234\" style=\"display:none;\">java.lang.Throwable</pre><br>"
-                        + "<a onclick=\"attachment=document.getElementById('embedding-1234'); attachment.style.display = (attachment.style.display == 'none' ? 'block' : 'none');return false\" href=\"#\">Attachment 4 (HTML text)</a><span id=\"embedding-1234\" style=\"display:none;\"><i>Hello</i> <b>World!<b></span><br><span style=\"color:red\">Attachment number 4, has unsupported mimetype 'unknown'.<br>File the bug <a href=\"https://github.com/damianszczepanik/cucumber-reporting/issues\">here</a> so support will be added!</span><br>"));
+                        + "<a onclick=\"attachment=document.getElementById('embedding-1234'); attachment.style.display = (attachment.style.display == 'none' ? 'block' : 'none');return false\" href=\"#\">Attachment 3 (image)</a><a href=\"data:image/jpeg;base64,/9j/4BBKRklGAQEBAf/bhAYEBQYFBAYGBQYHBwYIDQoQDQoNCgkJDQoUDg8MEBcUGBgXFBYWGh0lHxobIxwWFiAsICMmJykqKRkfLTAtKDAlKCkoAQcHBw0KCA0KEw0KDQoTKBoWGigoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgo/8ARCAEBAwERAhEBAxEB/8QBogEFAQEBAQEBAQIDBAUGBwgJDQoLEAIBAwMCBAMFBQQEAX0BAgMEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJDQoWFxgZGiUmJygpKjQ1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4eLj5OXm5+jp6vHy8/T19vf4+foBAwEBAQEBAQEBAQECAwQFBgcICQ0KCxECAQIEBAMEBwUEBAECdwECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRDQoWJDThJfEXGBkaJicoKSo1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoKDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uLj5OXm5+jp6vLz9PX29/j5+v/aDAMBAhEDET/5au7me8uprq8mlnuZnaSWWVyzyOxyWYnkkkkkmv//2Q==\"><img id=\"embedding-1234\" src=\"data:image/jpeg;base64,/9j/4BBKRklGAQEBAf/bhAYEBQYFBAYGBQYHBwYIDQoQDQoNCgkJDQoUDg8MEBcUGBgXFBYWGh0lHxobIxwWFiAsICMmJykqKRkfLTAtKDAlKCkoAQcHBw0KCA0KEw0KDQoTKBoWGigoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgo/8ARCAEBAwERAhEBAxEB/8QBogEFAQEBAQEBAQIDBAUGBwgJDQoLEAIBAwMCBAMFBQQEAX0BAgMEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJDQoWFxgZGiUmJygpKjQ1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoOEhYaHiImKkpOUlZaXmJmaoqOkpaanqKmqsrO0tba3uLm6wsPExcbHyMnK0tPU1dbX2Nna4eLj5OXm5+jp6vHy8/T19vf4+foBAwEBAQEBAQEBAQECAwQFBgcICQ0KCxECAQIEBAMEBwUEBAECdwECAxEEBSExBhJBUQdhcRMiMoEIFEKRobHBCSMzUvAVYnLRDQoWJDThJfEXGBkaJicoKSo1Njc4OTpDREVGR0hJSlNUVVZXWFlaY2RlZmdoaWpzdHV2d3h5eoKDhIWGh4iJipKTlJWWl5iZmqKjpKWmp6ipqrKztLW2t7i5usLDxMXGx8jJytLT1NXW19jZ2uLj5OXm5+jp6vLz9PX29/j5+v/aDAMBAhEDET/5au7me8uprq8mlnuZnaSWWVyzyOxyWYnkkkkkmv//2Q==\" style=\"display:none;\"/></a></br>"
```

Comment:
I've tried to encode this image via http://base64image.org/ but looks like this is not valid base64 array. - can you recheck?